### PR TITLE
more templates

### PR DESCRIPTION
**Pull request by Julian-Chu**
Hi @philipgriggs ,
Thank you so much for this go-postfix-templates. It's very useful in my daily coding life with Goland.
Here I also have some own postfix templates, would like to integrate to your repository.   Could you review the pull request?

### CODE REVIEW

**Comment by philipgriggs** [comment]
Thanks for the contribution and your feedback 👍 I'd like to include more templates in the repo, so please create another PR if you have any more ideas that would be useful.

File: http.postifxTemplates [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## for http development ##
+.handler : generate method for HandleFunc
```
Comment (philipgriggs) [inline http.postifxTemplates]: This example would work as a `GoLand` live template (https://www.jetbrains.com/help/go/using-live-templates.html). Do you have a good reason for including it in `custom-postfix-templates` instead? I'd consider adding it to `custom-postfix-templates` if there is an advantage.

Let me know if you're not familiar with how the live templates work.

File: http.postifxTemplates [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    ANY → func $expr$(w http.ResponseWriter, r *http.Request) {\
+                  $END$\
+             }\
+
+.handlerr : generate method for HandleFunc with receiver
```
Comment (philipgriggs) [inline http.postifxTemplates]: same as above

File: testing.postfixTemplates [hunk 3]
```diff
@@ -20,3 +20,18 @@
 
 .assertLen : assert length
     NIL      →  assert.Len(t, $expr$, $VAR0$, "$end$")
+
+
+## create tase case ##
+
+.test : generate test case
```
Comment (philipgriggs) [inline testing.postfixTemplates]: there is already a live template for `test` (eg If you type `test` somewhere in the editor). Again, is there an advantage to include it here as well?

File: testing.postfixTemplates [hunk 4]
```diff
@@ -20,3 +20,18 @@
...
+    ANY  →  func Test$expr$(t *testing.T) {\
+                  $END$\
+               }
+
+.bench : generate benchmark case
```
Comment (philipgriggs) [inline testing.postfixTemplates]: as above, there is already a live template for `bench`

File: http.postifxTemplates (hunk 1 above)
Comment (Julian-Chu) [reply http.postifxTemplates]: @philipgriggs , 

Thanks for your comments. I also have  some live templates with postfix templates format for my coding habit. But I think you opinion make sense to keep clear postfix templates. I delete them from this repository,  only just keep in my user-defined postfix.

File: general.postfixTemplates [hunk 5]
```diff
@@ -0,0 +1,7 @@
+## general use ##
```
Comment (philipgriggs) [inline general.postfixTemplates]: can you move these to `misc.postfixTemplates`? My idea was to include general templates in that file 
https://github.com/philipgriggs/go-postfix-templates/blob/master/misc.postfixTemplates

**Review by philipgriggs — CHANGES_REQUESTED** [review]

File: general.postfixTemplates (hunk 5 above)
Comment (Julian-Chu) [reply general.postfixTemplates]: done!

**Review by philipgriggs — APPROVED** [review]

**Comment by Julian-Chu** [comment]
hi @philipgriggs ,  just reminding,  I don't have write access to this repo, so can't merge it by myself.

**Pull request merged**